crates/router/src/connector/worldline/transformers.rs

```diff
@@ -430,3 +474,28 @@ pub struct ErrorResponse {
     pub error_id: Option<String>,
     pub errors: Vec<Error>,
 }
+
+#[derive(Debug, Eq, Hash, PartialEq)]
+pub enum CardProduct {
+    AmericanExpress,
+    Jcb,
+    Maestro,
+    Master,
+    UnionPay,
+    Visa,
+    Discover,
+}
+
+impl CardProduct {
+    fn value(&self) -> i16 {
```

Comment:
1. Please name this method as `product_id()` or something similar, to provide information on what value is being returned here.
2. Do you need the return type to be `i16`? From what I can see, the maximum value being returned is 128, so a `u8` should suffice. If you need return values greater than 255, then use `u16`, otherwise use `u8`.